test/tools/libtesteth/FillJsonFunctions.cpp

```diff
@@ -52,19 +52,19 @@ json_spirit::mObject fillJsonWithTransaction(Transaction const& _txn)
 	return txObject;
 }
 
-json_spirit::mObject fillJsonWithStateChange(State const& _stateOrig, eth::State const& _statePost, std::vector<dev::eth::detail::Change> const& _changeLog)
+json_spirit::mObject fillJsonWithStateChange(State const& _stateOrig, eth::State const& _statePost, std::vector<dev::eth::Change> const& _changeLog)
 {
 	json_spirit::mObject oState;
 	if (!_changeLog.size())
 		return oState;
 
-	using changeKind = eth::detail::Change::Kind;
-	using changeStruct = eth::detail::Change;
-	eth::detail::ChangeLog changeLog = _changeLog;
+	using ChangeKind = eth::Change::Kind;
+	using ChangeStruct = eth::Change;
+	eth::ChangeLog changeLog = _changeLog;
 
 	//Sort the vector by address field
 	std::sort(changeLog.begin(), changeLog.end(),
-		[ ]( const changeStruct& lhs, const changeStruct& rhs )
+		[ ]( const ChangeStruct& lhs, const ChangeStruct& rhs )
```

Comment:
Too much spaces here. Should be: `[](const ChangeStruct& lhs, const ChangeStruct& rhs)`.